qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -5,35 +5,24 @@
 #
 
 from test_framework.test_framework import ComparisonTestFramework
-from test_framework.util import *
+from test_framework.util import start_nodes
```

Comment:
These changes are going to make merging upstream's migration to Python 3 annoying. But then, so will the functional changes, so meh.